handle/error.go

```diff
@@ -0,0 +1,21 @@
+package handle
+
+import (
+	"context"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+)
+
+// Error is a wrapper around Abandon() that allows to trace the error before abandoning the message
```

Comment:
Should we just always force Abandon to take an error and have one less handle? I'm trying to think of a case where I want to abandon but can't be bothered to say why with an error.New

Reply:
If you don't want to handle the message, you just complete it.
That happens if you only want to handle a subset of messages. That's not an error.

Reply:
Error is what you'd use. I didnt want to force the creation of an error.
The original api to abandon doesn't have an error, I didn't see a reason to not expose it, if you prefer to handle and trace the error yourself.